test/unit-tests/GovernanceTimelock.t.sol

```diff
@@ -0,0 +1,268 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.9;
+
+/// library imports
+import {Vm} from "forge-std/Test.sol";
+
+/// local imports
+import "../Setup.t.sol";
+import "src/libraries/Error.sol";
+import {GovernanceTimelock} from "src/controllers/GovernanceTimelock.sol";
+
+contract GovernanceTimelockTest is Setup {
+    event TransactionScheduled(uint256 indexed txId, address target, uint256 value, bytes data, uint256 eta);
+    event TransactionExecuted(uint256 indexed txId, address target, uint256 value, bytes data, uint256 eta);
+
+    event ExecutionPeriodUpdated(uint256 oldPeriod, uint256 newPeriod);
+    event DelayUpdated(uint256 oldDelay, uint256 newDelay);
+    event AdminUpdated(address oldAdmin, address newAdmin);
+
+    uint256 constant SRC_CHAIN_ID = 1;
```

Comment:
So that we clean up related things as we go along, can we move the `SRC_CHAIN_ID` constant up to `Setup.sol`. There are a few places in `Setup.sol` where we just directly use the etherum chain id (i.e. 1), that would be good to replace with this constant. We can also remove the separate declaration of the constant in `MultiMessageReceiver` and `MultiMessageSender`

It might also be good to have chain ID constants for the different chains (e.g. `POLYGON_CHAIN_ID = 137`), so references to them are clearer

Reply:
Done.